index.html

```diff
@@ -5,7 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>my portfolio</title>
-    <link rel="stylesheet" href="styling.css">
+    <link rel="stylesheet" href="styling.css" media="only screen and (max-width: 768px)">
+    <link rel="stylesheet" type="text/css" href="style.css" media="only screen and (min-width: 768px)">
```

Comment:
- [ ] You did a very great job to ensure that the media query for min-width is applied to have a desktop version of your portfolio. However, I kindly suggest considering implementing this in a CSS file instead of HTML for [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) ☺️.